test/amqqueue_SUITE.erl

```diff
@@ -27,50 +28,84 @@
 -define(QUEUE, <<"TestQueue">>).
 -define(DESTINATION, "/amq/queue/TestQueue").
 
-all_tests() ->
-    [[ok = run_test(TestFun, Version)
-      || TestFun <- [fun test_publish_no_dest_error/3,
-                     fun test_publish_unauthorized_error/3,
-                     fun test_subscribe_error/3,
-                     fun test_subscribe/3,
-                     fun test_unsubscribe_ack/3,
-                     fun test_subscribe_ack/3,
-                     fun test_send/3,
-                     fun test_delete_queue_subscribe/3,
-                     fun test_temp_destination_queue/3,
-                     fun test_temp_destination_in_send/3,
-                     fun test_blank_destination_in_send/3
-                     ]]
-     || Version <- ?SUPPORTED_VERSIONS],
-    ok.
-
-run_test(TestFun, Version) ->
-    {ok, Connection} = amqp_connection:start(#amqp_params_direct{}),
+all() ->
+    [{group, list_to_atom("version_" ++ V)} || V <- ?SUPPORTED_VERSIONS].
+
+groups() ->
+    Tests = [
+        publish_no_dest_error,
+        publish_unauthorized_error,
+        subscribe_error,
+        subscribe,
+        unsubscribe_ack,
+        subscribe_ack,
+        send,
+        delete_queue_subscribe,
+        temp_destination_queue,
+        temp_destination_in_send,
+        blank_destination_in_send
+    ],
+
+    [{list_to_atom("version_" ++ V), [sequence], Tests} 
+     || V <- ?SUPPORTED_VERSIONS].
+
+init_per_suite(Config) ->
+    rabbit_ct_helpers:log_environment(),
+    rabbit_ct_helpers:run_setup_steps(Config, 
+                                      rabbit_ct_broker_helpers:setup_steps()).
```

Comment:
I suggest you configure a nodename suffix in all testsuites, like here:
https://github.com/rabbitmq/rabbitmq-lvc-plugin/blob/master/test/lvc_SUITE.erl#L43-L45